### ISSUE TITLE
App not building issue after installing the mobule

### ISSUE THREAD

**Issue by NimeshMadhushka** [TheWidlarzGroup/react-native-video#2791]
# Bug

  I am getting the following error when run the application after installing the module

> Could not resolve all task dependencies for configuration ':app:debugRuntimeClasspath'.
   > Could not find com.yqritc:android-scalablevideoview:1.0.4.
     Searched in the following locations:
       - file:/C:/work/sample vidio play app/VideoPlayApp/node_modules/react-native/android/com/yqritc/android-scalablevideoview/1.0.4/android-scalablevideoview-1.0.4.pom
       - file:/C:/work/sample vidio play app/VideoPlayApp/node_modules/jsc-android/dist/com/yqritc/android-scalablevideoview/1.0.4/android-scalablevideoview-1.0.4.pom
       - https://repo.maven.apache.org/maven2/com/yqritc/android-scalablevideoview/1.0.4/android-scalablevideoview-1.0.4.pom
       - https://dl.google.com/dl/android/maven2/com/yqritc/android-scalablevideoview/1.0.4/android-scalablevideoview-1.0.4.pom
       - https://www.jitpack.io/com/yqritc/android-scalablevideoview/1.0.4/android-scalablevideoview-1.0.4.pom
     Required by:
         project :app > project :react-native-video

## Platform

  Android

## Environment info

React native info output:

```bashSystem:
    OS: Windows 10 10.0.22000
    CPU: (8) x64 Intel(R) Core(TM) i5-10210U CPU @ 1.60GHz
    Memory: 3.17 GB / 15.84 GB
  Binaries:
    Node: 16.15.0 - C:\Program Files\nodejs\node.EXE
    Yarn: 1.22.19 - ~\AppData\Roaming\npm\yarn.CMD
    npm: 7.20.3 - C:\Program Files\nodejs\npm.CMD
    Watchman: Not Found
  SDKs:
    Android SDK: Not Found
    Windows SDK:
      AllowAllTrustedApps: Disabled
  IDEs:
    Android Studio: Version     2020.3.0.0 AI-203.7717.56.2031.7935034
    Visual Studio: Not Found
  Languages:
    Java: 11.0.15 - C:\Program Files\Common Files\Oracle\Java\javapath\javac.EXE
  npmPackages:
    @react-native-community/cli: Not Found
    react: 18.0.0 => 18.0.0
    react-native: 0.69.3 => 0.69.3
    react-native-windows: Not Found
```

Library version: 5.2.0

## Steps To Reproduce

1. yarn add react-native-video
2. npx react-native run-android

## Expected behaviour

1. open the application

## Video sample
If possible, include a link to the video that has the problem that can be streamed or downloaded from.

**Comment by freeboub** [comment]
Can you test this PR please ? https://github.com/react-native-video/react-native-video/pull/2795
It should fix this issue on 5.X version

**Comment by freeboub** [comment]
change released in 5.21.
